frontend/src/common/colors.js

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
if we will structure our app based on features, as you started with the Troops folder (which is a great way I think!), then there is no need for `common` and `components` folder, only the feature folders, and one common folder, which should contain the generic stuff, for example the styles. into the styles you can put the colors and the styles into one file, no need for separating them into 2 files.

Reply:
I think it's not a good SRP structure if we put colors and styles into one file.
I just want to use common colors like this...
```js
import Colors from '../common/colors';

const styles = {
  color: Colors.red,
}
```

Reply:
If you think it's better this way, I can accept this, separate colors and styles:)

But what about the 'component' folder? if we split the app based on the features (Troops, Buildings, etc. folders), then the components will go in the feature folders. This way a component folder in the src can be misleading. We could put the generic components folder into the common folder as well. Then everything generic would be in the common folder, common colors, styles, components: a styles folder with the styles and colors, and a component folder with the Cards for example.  

This is what I'm talking about:

src
  - common
     - styles
       - colors.js
       - styles.js
     - components
       - Cardview.js
  - Troop
  - Building
  ... etc.

Reply:
nice suggestion about that, I will follow it :-D

Reply:
I'm glad you agree, thanks :)